Helper/Processor.php

```diff
@@ -33,12 +34,13 @@ public function __construct(UrlGeneratorInterface $router, TranslatorInterface $
     /**
      * Generates pagination template data
      *
-     * @param array $queryParams
-     * @param array $viewParams
+     * @param SlidingPagination $pagination
+     * @param array             $queryParams
+     * @param array             $viewParams
      *
      * @return array
      */
-    public function render($pagination, array $queryParams = array(), array $viewParams = array())
+    public function render(SlidingPagination $pagination, array $queryParams = array(), array $viewParams = array())
```

Comment:
this does not typehint the interface coming from knp-components, because the implementation works only with the implementation provided in the bundle (which has more methods)